gg/tests/test_db.py

```diff
@@ -25,5 +27,33 @@ def test_existence():
     docs = list_from_db(test=True)
     assert len(docs) == 1
 
-    name = docs[0]["Name"]
+    name = docs[0]["name"]
     assert name == "TEST"
+
+    static_route = docs[0]["routes"]["Static"]
+    assert static_route == "https://gg-scraper-example.now.sh/static/"
+
+
+def test_rejection():
+    # Clear the testing collection
+    DELETE_ALL_PLEASE_ONLY_USE_THIS_FOR_TESTING(test=True)
+
+    name = "TEST"
+    url = "https://gg-scraper-example.now.sh"
+    namesList = ["Routes", "Test1", "Data", "Static"]
+    routesList = [
+        "https://gg-scraper-example.now.sh/routes",
```

Comment:
Can we use the url instead of hard coding the route list. This should make it easier if we want to test other endpoints later on.